src/index.html

```diff
@@ -2,11 +2,331 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/reset.css"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="header-container">
+      <header class="header">
+        <a
+          href="index.html"
+          class="logo"
+        >
+          <img
+            class="logo_img"
+            src="./images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list list">
```

Comment:
you don't have styles for `list` block - you don't need it
`nav__list` - as a child of `nav` block will be enough here

the same for items :arrow_down:

Reply:
i need step-by-step instruction in screenshots for this links))
i m sorry, so stupid))

Reply:
@ratuh_hokage telega